### DISCUSSION TITLE
20210911 Issues

### DISCUSSION THREAD

**Discussion by sibezwo** [rern/rAudio#513] [Fixed Issues]
Hello Rern

On my RPi3B with the current version 20210911 I discovered some bugs:

- Date and time are not synchronized. After a reset the clock always starts at 2021-08-31 15:13:30 (Timezone: Europe - Zurich +02:00).

- The values ​​from the Sound Profile are not set correctly after a reset.

- Typo in System - Status: CPU Temperatue instead of Temperature.

- The volume dial and buttons are not exactly in the middle, but shifted to the right (at least on my Pixel 4 with Chrome browser).

- In the Playback settings, Coverart should only be able to be disabled if Top-Bottom Bars AND Bars always on are enabled. Otherwise you don't have a chance to get into the menu and you are lost!

Thomas

**Comment by rern** [comment]
> * Date and time are not synchronized. After a reset the clock always starts at 2021-08-31 15:13:30 (Timezone: Europe - Zurich +02:00).

Try: https://github.com/rern/rAudio-1/discussions/472#discussioncomment-1312970 

> * The volume dial and buttons are not exactly in the middle, but shifted to the right (at least on my Pixel 4 with Chrome browser).

Post your screenshot.

> * The values ​​from the Sound Profile are not set correctly after a reset.
> * Typo in System - Status: CPU Temperatue instead of Temperature.
> ...
> * In the Playback settings, Coverart should only be able to be disabled if Top-Bottom Bars AND Bars always on are enabled. Otherwise you don't have a chance to get into the menu and you are lost!

To be fixed by next update.

**Reply by ceejayemm** [reply]
I can confirm getting the date/time set on a RPi3B+ is still problematical. The fix in #472 doesn't work for me.

Chris

**Comment by sibezwo** [comment]
Sorry for the late reply but I had serious back pain.
Here's the screenshot:

<img src="https://user-images.githubusercontent.com/78496250/134007282-c2a958f1-df33-40fa-aeed-12661f998e35.png" width=300>

**Comment by johnsoca2** [comment]
After the upgrade to 20210911, I cannot access Settings - Player.  When I select them from the menu, the rAudio logo just flashes in the middle of the screen.  I can use the back browser button to return to my previous screen, but nothing in Player Settings.  I tried resetting the environment, but having the same problem.  Any ideas?  Thanks.

**Reply by rern** [reply]
Let me see the output of SSH command line:
`player-data.sh | jq .`

**Comment by chipdipru** [comment]
After the upgrade to 20210925, I cannot access Settings - System. When I select them from the menu I see an error - **Errors: Unexpected token , in JSON at position 1287**
updated through wifi
Then

![errors2](https://user-images.githubusercontent.com/43340836/134798912-567e8132-e826-4328-a1d0-127ce40bb952.jpg)

**Reply by rern** [reply]
Try:
`rm -f /srv/http/data/system/soundprofile*`

**Comment by chipdipru** [comment]
Hi rern
I deleted the file soundprofile*
The error remained. everything else works well.
have another idea?
![aradio1](https://user-images.githubusercontent.com/43340836/134807669-bf0dfe5a-0dce-4851-ad1f-75dc0faee375.jpg)

**Reply by rern** [reply]
Let's see the missing values:
```
sysctl kernel.sched_latency_ns | awk '{print $NF}' | tr -d '\0'
sysctl vm.swappiness | awk '{print $NF}'
ifconfig eth0 | awk '/mtu/ {print $NF}'
ifconfig eth0 | awk '/txqueuelen/ {print $4}'
```

**Comment by chipdipru** [comment]
sysctl kernel.sched_latency_ns | awk '{print $NF}' | tr -d '\0'
18000000
sysctl vm.swappiness | awk '{print $NF}'
60
ifconfig eth0 | awk '/mtu/ {print $NF}'
eth0: error fetching interface information: Device not found
ifconfig eth0 | awk '/txqueuelen/ {print $4}'
eth0: error fetching interface information: Device not found

Raspberry Pi 3 Model A+

**Reply by rern** [reply]
Fixed by today's update.

**Reply by chipdipru** [reply]
thank you very much! now there are no errors!
